### PR TITLE
COM-414 Active patient  list - only show patients for the selected lo…

### PR DESCRIPTION
**Pull request by prashantsable1991**
Updated location.csv
Updated visit types
Updated global properties
Updated entity mapping sql

### CODE REVIEW

**Comment by tmvumbi2** [comment]
@prashantsable1991 Please also retire the location 'OT'
image

File: configuration/locations/locations.csv [hunk 1]
```diff
@@ -41,9 +36,5 @@ bb0e512e-d225-11e4-9c67-080027b662ec,TRUE,Labour Ward,,,,,,,,,,,,,,,
...
 e48fb2b3-d490-11e5-b193-0800270d80ce,TRUE,General Ward - Room 2,,,,,,,,,,,,,,,
 8d6c993e-c2cc-11de-8d13-0010c6dffd0f,TRUE,Unknown Location,,,,,,,,,,,,,,,
-0d9bf0c2-39f5-489a-a660-927f1ac97b7b,,LOCATION_CARDIOLOGY,LOCATION_CARDIOLOGY,,Login Location; Visit Location; Appointment Location,,,,,,,,,,,,
-c6e23237-5a0d-438a-9381-ac403d30054f,,LOCATION_DELIVERY,LOCATION_DELIVERY,,Login Location; Visit Location; Appointment Location,,,,,,,,,,,,
-79157d5d-1e45-407d-bf15-ae8dc1674a70,,LOCATION_DENTISTRY,LOCATION_DENTISTRY,,Login Location; Visit Location; Appointment Location,,,,,,,,,,,,
-8568c575-16fc-4330-a3bb-c0460c15a298,,LOCATION_GYNAECOLOGY,LOCATION_GYNAECOLOGY,,Login Location; Visit Location; Appointment Location,,,,,,,,,,,,
-df204697-a6c6-4768-a4e8-7082940fc8eb,,LOCATION_OPTHALMOLOGY,LOCATION_OPTHALMOLOGY,,Login Location; Visit Location; Appointment Location,,,,,,,,,,,,
-4dbf4014-4f2c-4c74-ad3e-2f6df3f60c5d,,LOCATION_RADIOLOGY,LOCATION_RADIOLOGY,,Login Location; Visit Location; Appointment Location,,,,,,,,,,,,
+
+
```
Comment (tmvumbi2) [inline configuration/locations/locations.csv]: @prashantsable1991 Please delete the line 31 `c5854fd7-3f12-11e4-adec-0800271c1b75,TRUE,Registration Desk,,,,,,,,,,,,,,,`: this uuid is used by the new registration desk location (see line 25) and will be overwritten

File: sql/entity_mapping.sql [hunk 2]
```diff
@@ -1,16 +1,26 @@
...
-    '710e6838-806a-11e9-bc42-526af7764f64',
-    '710e6964-806a-11e9-bc42-526af7764f64',
-    '710e6d1a-806a-11e9-bc42-526af7764f64',
-    '710e6e6e-806a-11e9-bc42-526af7764f64',
-    '710e6f9a-806a-11e9-bc42-526af7764f64',
-    '710e713e-806a-11e9-bc42-526af7764f64'
-);
\ No newline at end of file
+INSERT IGNORE INTO entity_mapping
+    (uuid,entity_mapping_type_id,entity1_uuid,entity2_uuid)
```
Comment (tmvumbi2) [inline sql/entity_mapping.sql]: @prashantsable1991 please add the column 'date_created', the value can be set to 'now()'

File: configuration/locations/locations.csv [hunk 3]
```diff
@@ -1,34 +1,29 @@
...
+ee0644c1-53e1-4c8c-bcd9-2308c608be60, ,LOCATION_LABORATORY,LOCATION_LABORATORY,LOCATION_HOSPITAL,Login Location;Appointment Location,,,,,,,,CAMEROON,,,,
+ff6b443b-958b-4cf9-b187-d0f6da6b6d94, ,LOCATION_MAJOR_SURGERY,LOCATION_MAJOR_SURGERY,LOCATION_HOSPITAL,Login Location;Appointment Location,,,,,,,,CAMEROON,,,,
+0d414ccb-55a2-480f-93ff-c2af7061f0ba, ,LOCATION_MATERNITY,LOCATION_MATERNITY,LOCATION_HOSPITAL,Login Location;Appointment Location,,,,,,,,CAMEROON,,,,
+0607712c-aced-44da-8534-72d639bd9994, ,LOCATION_MINOR_SURGERY,LOCATION_MINOR_SURGERY,LOCATION_HOSPITAL,Login Location;Appointment Location,,,,,,,,CAMEROON,,,,
+4863dd00-0543-40c1-9cb0-f019c11e86bf, ,LOCATION_NEONATOLOGY,LOCATION_NEONATOLOGY,LOCATION_HOSPITAL,Login Location;Appointment Location,,,,,,,,CAMEROON,,,,
+4dd3bdc2-38ab-450e-b36f-784ed6f8427d, ,LOCATION_OPD,LOCATION_OPD,LOCATION_HOSPITAL,Login Location;Appointment Location,,,,,,,,CAMEROON,,,,
+df204697-a6c6-4768-a4e8-7082940fc8eb, ,LOCATION_OPTHALMOLOGY,LOCATION_OPTHALMOLOGY,LOCATION_HOSPITAL,Login Location;Appointment Location,,,,,,,,CAMEROON,,,,
```
Comment (tmvumbi2) [inline configuration/locations/locations.csv]: @prashantsable1991 There is a small typo here, the right spelling is:  OPHTHALMOLOGY

File: configuration/locations/locations.csv [hunk 4]
```diff
@@ -41,9 +36,5 @@ bb0e512e-d225-11e4-9c67-080027b662ec,TRUE,Labour Ward,,,,,,,,,,,,,,,
 bb0eb071-d225-11e4-9c67-080027b662ec,TRUE,Labour Ward - 1,,,,,,,,,,,,,,,
 e48fb2b3-d490-11e5-b193-0800270d80ce,TRUE,General Ward - Room 2,,,,,,,,,,,,,,,
 8d6c993e-c2cc-11de-8d13-0010c6dffd0f,TRUE,Unknown Location,,,,,,,,,,,,,,,
-0d9bf0c2-39f5-489a-a660-927f1ac97b7b,,LOCATION_CARDIOLOGY,LOCATION_CARDIOLOGY,,Login Location; Visit Location; Appointment Location,,,,,,,,,,,,
-c6e23237-5a0d-438a-9381-ac403d30054f,,LOCATION_DELIVERY,LOCATION_DELIVERY,,Login Location; Visit Location; Appointment Location,,,,,,,,,,,,
-79157d5d-1e45-407d-bf15-ae8dc1674a70,,LOCATION_DENTISTRY,LOCATION_DENTISTRY,,Login Location; Visit Location; Appointment Location,,,,,,,,,,,,
-8568c575-16fc-4330-a3bb-c0460c15a298,,LOCATION_GYNAECOLOGY,LOCATION_GYNAECOLOGY,,Login Location; Visit Location; Appointment Location,,,,,,,,,,,,
-df204697-a6c6-4768-a4e8-7082940fc8eb,,LOCATION_OPTHALMOLOGY,LOCATION_OPTHALMOLOGY,,Login Location; Visit Location; Appointment Location,,,,,,,,,,,,
-4dbf4014-4f2c-4c74-ad3e-2f6df3f60c5d,,LOCATION_RADIOLOGY,LOCATION_RADIOLOGY,,Login Location; Visit Location; Appointment Location,,,,,,,,,,,,
+
```
Comment (tmvumbi2) [inline configuration/locations/locations.csv]: @prashantsable1991 please remove the 2 empty lines, this is causing errors in the back-end log:
```
ERROR - CsvParser.saveAll(184) |2019-07-09 09:34:43,501| An OpenMRS object could not be constructed or saved from the following CSV line: [null]
ERROR - CsvParser.saveAll(184) |2019-07-09 09:34:43,504| An OpenMRS object could not be constructed or saved from the following CSV line: [null]
```

File: configuration/globalproperties/activePatients.xml [hunk 5]
```diff
@@ -0,0 +1,34 @@
+<config>
```
Comment (tmvumbi2) [inline configuration/globalproperties/activePatients.xml:1]: @prashantsable1991 please delete the file `activePatientsByLocation.xml` that is updating the same global property

**Comment by prashantsable1991** [comment]
@tmvumbi2 Thanks for the review. Please review once.

**Pull request merged**